app/templates/_base_page.html

```diff
@@ -32,10 +32,9 @@
         "html": 'Help us improve the Digital Marketplace - <a class="govuk-link" href="'  + url_for('external.help') + '">send your feedback</a>'
       }) }}
       {% block breadcrumbs %}{% endblock %}
+      {% include "toolkit/flash_messages.html" %}
```

Comment:
This is different to the other apps, where we left the flash messages inside the main. Is this change needed? 🤔

Reply:
This is needed for the moment otherwise the flash message is pushed down a bit further than whats needed.

I agree, they should be identical but they apps are in different states at the moment, eg. not using govuk-grid or other sass mixins.

Reply:
That's interesting; so this fix will become necessary for the other apps later on? What would happen if we applied this fix to the other apps now?

Reply:
I don't want to spend any more time on this issue as it is a temporary fix and the bigger issue is trying to fix notifications/temp/flash banners and will have to revisit this again.